docs/landing/data/releases.toml

```diff
@@ -16,78 +21,11 @@ current = "3.12.0"
   docs = "./3.10"
   api = "./3.10/javadoc"
 
-[[versions]]
-  version = "3.9.0"
-  docs = "./3.9"
-  api = "./3.9/javadoc"
-
-[[versions]]
-  version = "3.8.2"
-  docs = "./3.8"
-  api = "./3.8/javadoc"
-
-[[versions]]
-  version = "3.7.1"
-  docs = "./3.7"
-  api = "./3.7/javadoc"
-
-[[versions]]
-  version = "3.6.4"
-  docs = "./3.6"
-  api = "./3.6/javadoc"
-
-[[versions]]
-  version = "3.5.0"
-  docs = "./3.5"
-  api = "./3.5/javadoc"
-
-[[versions]]
-  version = "3.4.3"
-  docs = "./3.4"
-  api = "./3.4/javadoc"
-
-[[versions]]
-  version = "3.3.0"
-  docs = "./3.3"
-  api = "http://api.mongodb.com/java/3.3"
-
-[[versions]]
-  version = "3.2.2"
-  docs = "./3.2"
-  api = "http://api.mongodb.com/java/3.2"
-
-[[versions]]
-  version = "3.1.1"
-  docs = "./3.1"
-  api = "http://api.mongodb.com/java/3.1"
-
-[[versions]]
-  version = "3.0.4"
-  docs = "./3.0"
-  api = "http://api.mongodb.com/java/3.0"
-
 [[versions]]
```

Comment:
Do you think it might be time to ditch this one?

Reply:
Yeah :) - done